### DISCUSSION TITLE
New Pointer

### DISCUSSION THREAD

**Discussion by not-available-username** [organicmaps/organicmaps#2853] [Ideas]
![1](https://user-images.githubusercontent.com/92886370/176152266-ce22c805-1f49-4f28-8741-6a0ddb427cee.jpg)

I decided to open a new discussion in order to better track this thing, following this https://github.com/organicmaps/organicmaps/issues/1060#issuecomment-1092797946 and this https://github.com/organicmaps/organicmaps/issues/1060#issuecomment-1092859025 I tried to create a new pointer and error radius circle.

I agree that the current pointer is not always readable and, in my opinion, is also a bit outdated in terms of design, but I agree with @biodranik to not spend time with the support of different pointers and colours.

I think that with some minor changes the readability could be improved by a lot, essentially with a brighter colour and flat design.

I used #1f95f0 (it should be a colour already used in OM) for the pointer and for the GPS error radius (the circle has an opacity of 10% and the outline 20%). The corners of the pointer are rounded and there is a white outline to improve the contrast

Please let me know if this could work and I'll share the vector file!

### **Standard Theme**

![2](https://user-images.githubusercontent.com/92886370/176152288-3d5bdd79-8602-4f86-83d1-ec915f8d83fe.jpg)

### **Night Theme**

![3](https://user-images.githubusercontent.com/92886370/176152301-f49475dd-326a-4f08-92d5-2aed7483d040.jpg)

### **Color blindness** 

![4](https://user-images.githubusercontent.com/92886370/176152308-2f6eefae-bfd2-4689-8f86-a5ed345eb730.jpg)

**Comment by canonex** [comment]
I agree with the points made, and I think the design is clear and modern.

**Comment by not-available-username** [comment]
@biodranik in order to implement this change what is needed? I can provide the icon in an .svg format and the values for the colour and transparency 👍🏻

**Reply by biodranik** [reply]
Isn't there already a similar icon drawn in the navigation mode?

**Reply by not-available-username** [reply]
yes, is similar but following this https://github.com/organicmaps/organicmaps/issues/1060#issuecomment-1092797946 I made some improvements, I hope.

**Comment by biodranik** [comment]
Let's compare it with a current arrow side to side, and clarify what we want to improve/solve exactly. 

Flat design IMO doesn't make it easier to detect the direction of the arrow.

**Reply by not-available-username** [reply]
Here is the svg, if someone wants to do a side by side comparison. I want to make it more consistent with the overall graphic of the app and the location icon already used on the right side of the screen. A single colour with a white outline enhance the contrast, I cannot understand how a flat design arrow can be misunderstood, sorry.

![OM gps icon](https://user-images.githubusercontent.com/92886370/213940740-cce8edc2-beaf-4ca7-a760-182cef104441.svg)

**Comment by hb0nd** [comment]
i like this, my one concern would be perspective mode, would it still look good/be informative as a flat icon?

**Reply by not-available-username** [reply]
> i like this, my one concern would be perspective mode, would it still look good/be informative as a flat icon?

If you have Google Maps you can try how it looks in perspective, since it has a flat icon. Keep in mind that in perspective mode the arrow points always in front of you.

**Comment by rtsisyk** [comment]
I like this flat icon. Modern style. Current 3D icon went out of fashion already.

**Comment by biodranik** [comment]
Let's focus on usability, not on the "modern look". The new icon should be perfectly recognizable in dark/light mode, not too bright at night, distinguishable under the sunlight, with and without a perspective mode, and with good and not-good eyesight. Moreover, the direction it points to should be very clear.

How arrows look in other popular apps?

**Reply by not-available-username** [reply]
I prefer a cursor larger than the current one since my eyesight is not perfect

**Reply by biodranik** [reply]
There is a "larger font" option already in OM. We can use it to enlarge the cursor. An issue can be created for that.

**Comment by not-available-username** [comment]
Google https://i.redd.it/um1sjbba5qq21.jpg
Apple https://www.apple.com/newsroom/ea/images/product/apps/standard/Apple-Maps-update-2022-hero_inline.jpg.large.jpg
Waze https://images.sftcdn.net/images/t_app-cover-l,f_auto/p/174bedb0-99eb-11e6-aa38-00163ec9f5fa/2584289580/waze-1.png
OSMAnd https://osmand.net/assets/images/promo-5s-6178a8b1e73f733beb67f61b292b67e0.png

**Comment by d4f5409d** [comment]
> Let's focus on usability, not on the "modern look". The new icon should be perfectly recognizable in dark/light mode, not too bright at night, distinguishable under the sunlight, with and without a perspective mode, and with good and not-good eyesight. Moreover, the direction it points to should be very clear.
> 
> How arrows look in other popular apps?
> 

Side note: if we want to be competitive, we also should focus on design. I especially love this new icon idea. The Organic Maps exists for simplicity. I think  this new arrow perfectly outlines what we should follow in case of the app. On the accessibility line, I think it it isn't a big problem to distinguish it from the map itself.

**Reply by not-available-username** [reply]
> Side note: if we want to be competitive, we also should focus on design. I especially love this new icon idea. The Organic Maps exists for simplicity. I think this new arrow perfectly outlines what we should follow in case of the app. On the accessibility line, I think it it isn't a big problem to distinguish it from the map itself.

I've added an outline to have a proper contrast against the map. I agree, design is an important aspect of the UX, a good design is not only "nice" but has a better usability.

**Comment by d4f5409d** [comment]


**Reply by not-available-username** [reply]
> Do you have a matrix account? I'd get in touch with you for further messaging about this.

No, sorry. I have Discord...

**Comment by d4f5409d** [comment]
Would it be good for you, if I provided an obj/fbx file on this to just replace the current arrow right away?

**Comment by jumelles** [comment]
This is a great improvement!

**Comment by patepelo** [comment]
This proposal is very good and has gained a lot of support in this discussion 🥰 Should we convert it into an issue?

**Reply by not-available-username** [reply]
yes, please! [here](https://github.com/orgs/organicmaps/discussions/2853#discussioncomment-4751470) you can download the svg

**Comment by patepelo** [comment]
By the way, we could even consider having a dot like pointer when not navigating. What do you think?

Somethink like this is common:

![image](https://github.com/organicmaps/organicmaps/assets/23003127/95a2ffe6-1929-4f29-8c21-4afaf9853390)

**Reply by d4f5409d** [reply]
🇭🇺

**Reply by hb0nd** [reply]
a dot with an expanding field of view cone makes sense, as it's an intuitive way to indicate compass accuracy

**Comment by not-available-username** [comment]
Hello, any update on this topic?

**Comment by guplem** [comment]
Has any issue been created for this?

**Comment by Shringe** [comment]
I prefer the current design to the arrow over a flat one. It makes Organic Maps stand out.

**Reply by guplem** [reply]
Ideally it should be an option in settings...
